app/serializers/submission_serializer/sections/client_details.rb

```diff
@@ -7,10 +7,18 @@ def to_builder
 
           json.client_details do
             json.applicant Definitions::Applicant.generate(crime_application)
-            json.partner Definitions::Partner.generate(crime_application) if partner
+            json.partner partner_json
```

Comment:
I removed this in an earlier PR as part of a fix to not display partner details section if partner is not present but found that it broke rehydration so reverted and just opted to handle conditionally checking in the section